### PR TITLE
Tw 663 main info

### PR DESCRIPTION
**Pull request by herkoss**

### CODE REVIEW

**Review by keshan3262 — DISMISSED** [review]

**Review by keshan3262 — APPROVED** [review]

**Review by keshan3262 — APPROVED** [review]

**Review by keshan3262 — COMMENTED** [review]
It looks like changes don't refer to some comments from @alex-tsx

File: src/screens/earn/main-info/main-info.tsx [hunk 1]
```diff
@@ -66,7 +67,12 @@ export const MainInfo: FC = () => {
     return result;
   }, [farms]);
 
-  const areAnyRewards = totalClaimableRewardsInUsd.isGreaterThan(DEFAULT_AMOUNT);
+  const areAllRewardsClaimable = useMemo(() => {
```
Comment (keshan3262) [inline src/screens/earn/main-info/main-info.tsx:70]: As I understood, a user has to wait now for stakes across all farms to pass the due date, so they can't harvest full rewards in some cases

Comment (herkoss) [reply src/screens/earn/main-info/main-info.tsx:70]: Yep

**Review by keshan3262 — DISMISSED** [review]

**Review by keshan3262 — APPROVED** [review]

**Review by keshan3262 — APPROVED** [review]

**Review by keshan3262 — APPROVED** [review]

**Review by keshan3262 — APPROVED** [review]

**Review by keshan3262 — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_723

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_723

**Pull request merged**